### DISCUSSION TITLE
Dodatkowe informacje w zasobie /sale/quality

### DISCUSSION THREAD

**Discussion by shivson1337** [allegro/allegro-api#9721] [Suggestion - verification]
19 sierpnia 2024 został zaprezentowany nam zasób **/sale/quality**, który umożliwia pobranie danych o jakości naszej sprzedaży.

Na ten moment widzimy tam ogólną ocenę konta, ilośc zebranych punktów oraz ich maksymalną możliwą ilość z podziałem na różne kategorie.

Dobrze, że możemy teraz pobrać te dane przez API, brakuje niestety dodatkowego pola, które widoczne jest w panelu na Allegro.

Chodzi dokładniej o pole "Wartość", które prezentuje nam, w zależności od kategorii różne wartości (procentowe, tekstowe).

Wartości, o których mowa wyglądają mniej więcej tak:

> Szybka wysyłka, wartość: 47%
> Wysyłka w terminie, wartość: 98%
> Regulamin i zasady, wartość: 1 naruszenie regulaminu
> Średni czas odpowiedzi, wartość: 0 godz. 6 min.

Niestety sama ilość punktów nie pozwala na wyliczenie tych wartości, a to uniemożliwia śledzenie realnych postępów we wzroście lub spadku danego wskaźnika. Na ten moment pozostaje nam dalsze codzienne logowanie się na wiele kont i ręczne spisywanie tych wartości, w które niejednokrotnie wkrada się błąd ludzki zaburzając pogląd na sytuację.

Będziemy wdzięczni za rozpatrzenie propozycji i uwzględnienie jej w nadchodzących zmianach w API.

**Comment by MartaNowaczyk** [comment]
Sugestia przekazana do zespołu odpowiedzialnego za ten endpoint - zmieniłam status na "Suggestion - verification".
